### PR TITLE
📝Add Dobot Notebooks

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Description

- Add notebooks for Dobot

### CODE REVIEW

File: requirements/extras.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ gradio>=2.9.4
 tensorboard
 wandb==0.12.17
 GitPython
```
Comment (djdameln) [inline requirements/extras.txt:5]: Would it be an idea to remove this from Anomalibs requirements and add it as a prerequisite in the README of the notebooks instead?

Comment (ashwinvaidya17) [reply requirements/extras.txt:5]: I think these were moved https://github.com/openvinotoolkit/anomalib/pull/870/files#diff-3d0f7ff059ea9ec68b83f0d1a6eee038151708acf96afe054eb44a9d92786f80

Comment (ashwinvaidya17) [reply requirements/extras.txt:5]: Oh you mean GitPython. Yeah let's do that

Comment (ashwinvaidya17) [reply requirements/extras.txt:5]: How about requirements/notebooks.txt?

**Review by paularamo — APPROVED** [review]

**Pull request merged**